Mathlib/LinearAlgebra/FinsuppVectorSpace.lean

```diff
@@ -154,9 +154,15 @@ theorem _root_.Finset.sum_single_ite [Fintype n] (a : R) (i : n) :
   simp only [apply_ite (Finsupp.single _), Finsupp.single_zero, Finset.sum_ite_eq,
     if_pos (Finset.mem_univ _)]
 
-theorem equivFun_symm_stdBasis [Finite n] (b : Basis n R M) (i : n) :
-    b.equivFun.symm (LinearMap.stdBasis R (fun _ => R) i 1) = b i := by
+theorem equivFun_symm_single [Finite n] (b : Basis n R M) (i : n) :
+    b.equivFun.symm (LinearMap.single R (fun _ => R) i 1) = b i := by
```

Comment:
I think I'd argue this should be
```suggestion
    b.equivFun.symm (Pi.single i 1) = b i := by
```
which would then make it safe to be `simp` too.